js/site_nav.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/* global require, module, document */
+
+function SiteNav(selector) {
+  this.$body = $(selector);
+  this.$dropdowns = this.$body.find('ul ul');
```

Comment:
This variable is never used, but seems to be the same selector as `$subLists` in `assignAria`.